### DISCUSSION TITLE
Custom RPC Support

### DISCUSSION THREAD

**Discussion by swheel33** [WalletConnect/walletconnect-monorepo#4166] [Developer Support] [answered]
Is there a way to add custom rpcs for web3modal v3? I don't see any information on the docs and the discussion questions seem related to v2. Whenever I replace the defaultWagmiConfig with a custom one I lose all the connectors

**Comment by glitch-txs** [comment, marked as answer]
you should use the usual wagmi config instead of defaultWagmiConfig which is abstracted, we'll add this option with Wagmi v2 though

**Reply by swheel33** [reply]
How do you add all the web3modal connectors then? The only one that shows up is browser wallet when I use wagmi config

**Reply by glitch-txs** [reply]
check in the **custom** section in the documentation, it's explained in the getting started section
https://docs.walletconnect.com/web3modal/about

**Reply by swheel33** [reply]
Thanks so much!
